Applications/opensense/opensense.cpp

```diff
@@ -158,6 +183,11 @@ int main(int argc, char **argv)
                     return 0;
                 }
                 else if ((option == "-Setup") || (option == "-S")) {
+                    if (argc < 3) {
+                        cout << "Setup file .xml is expected after -S option. Please fix and retry." << endl;
```

Comment:
`A setup (.xml) file was expected but no file was provided. If no setup file exists, use the -PS option to print a default setup file that can be edited.`